src/components/MyPopup.js

```diff
@@ -22,7 +23,9 @@ export default function MyPopup({ selectedInfection, setSelectedInfection }) {
             </h3>
 
             <div id="data-render">
-                <DataRender
+                {/* <DataRender
+                    stateId={selectedInfection.id} /> */}
```

Comment:
Atomic changes - what is the purpose of keeping this and the import above commented?

You have git, so you can also revert commits and checkout the previous version, or look up the previous commit to see how it was done before.

Get into the habit of commit clean and complete code. This looks like you are about to revert it anytime.

Reply:
Fixed